my-account/app/myaccount/login/SignUpFormData.java

```diff
@@ -0,0 +1,88 @@
+package myaccount.login;
+
+import io.sphere.sdk.models.Base;
+import play.data.validation.Constraints;
+
+public class SignUpFormData extends Base {
+    @Constraints.Required
```

Comment:
It is a dropdown where you have a pre-selected value, so I put `Required` because it's practically true (i.e. it's not possible via the UI to put it optional). Also usually in shops they need to know the title to write all the emails and letters, so it's usually mandatory because of that.

But what do you mean with "the form is screwed up with salutation"?